include/board.c

```diff
@@ -1,33 +1,74 @@
-#include<stdio.h>
-#include "board.h"
+#include <stdio.h>
+#include"board.h"
 
-void init_board(struct Board *board) {
-    for (int i = 0; i < SIZE; i++) {
-        for (int j = 0; j < SIZE; j++) {
-            board->board[i][j] = EMPTY;
```

Comment:
I'd say let the already defined function be as they were before i.e. `init_board` and `print_board`.